docs/install_faq.md

```diff
@@ -0,0 +1,129 @@
+# Installation: Frequently Asked Questions
+
+This section tracks some of the more frequently encountered issues with installing
+or getting started with Helm.
+
+**We'd love your help** making this document better. To add, correct, or remove
+information, [file an issue](https://github.com/kubernetes/helm/issues) or
+send us a pull request.
+
+## Downloading
+
+I want to know more about my downloading options.
+
+**Q: I can't get to GitHub releases of the newest Helm. Where are they?**
+
+A: We no longer use GitHub releases. Binaries are now stored in a
+[GCS public bucket](http://storage.googleapis.com/kubernetes-helm/).
+
+**Q: Why aren't there Debian/Fedora/... native packages of Helm?**
+
+We'd love to provide these or point you toward a trusted provider. If you're
+interested in helping, we'd love it. This is how the Homebrew formula was
+started.
+
+**Q: Why do you provide a `curl ...|bash` script?**
+
+A: There is a script in our repository (`scripts/get`) that can be executed as
+a `curl ..|bash` script. The transfers are all protected by HTTPS, and the script
+does some auditing of the packages it fetches. However, the script has all the
+usual dangers of any shell script.
+
+We provide it because it is useful, but we suggest that users carefully read the
+script first. What we'd really like, though, are better packaged releases of
+Helm.
+
+## Installing
+
+I'm trying to install Helm/Tiller, but something is not right.
+
+**Q: How do I put the Helm client files somewhere other than ~/.helm?**
+
+Set the `$HELM_HOME` environment variable, and then run `helm init`:
+
+```console
+export HELM_HOME=/some/path
+helm init --client-only
+```
+
+Note that if you have existing repositories, you will need to re-add them
+with `helm repo add...`.
+
+**Q: How do I install Helm locally, but not remotely?**
```

Comment:
not sure if we need to communicate this differently.  It is not installing helm locally, it is setting up client side configuration.